indico/modules/events/registration/badges.py

```diff
@@ -96,8 +96,10 @@ def _draw_badge(self, canvas, registration, template, tpl_data, pos_x, pos_y):
                     text = placeholder.render(registration)
                 elif placeholder.group == 'event':
                     text = placeholder.render(registration.event)
-                else:
+                elif placeholder.group == 'fixed':
                     text = placeholder.render(item)
+                else:
+                    raise ValueError('Unknown placeholder group: `{}`'.format(placeholder.group))
```

Comment:
Don't you have the pre-commit.githook enabled? pyupgrade would point out that this should use an f-string:

```suggestion
                    raise ValueError(f'Unknown placeholder group: {placeholder.group}')
```

Reply:
Okay, I will use it for subsequent PRs. Thank you for the reviews.